Tools/peg_generator/.ruff.toml

```
@@ -0,0 +1,35 @@
+extend = "../../.ruff.toml"  # Inherit the project-wide settings
+extend-exclude = [
+    # Generated files:
+    "Tools/peg_generator/pegen/grammar_parser.py",
+]
+
+[lint]
+select = [
+    "F",  # Enable all pyflakes rules
+    "I",  # Enable all isort rules
+    "UP",  # Enable all pyupgrade rules by default
+    "RUF100",  # Ban unused `# noqa` comments
+    "PGH004",  # Ban blanket `# noqa` comments (only ignore specific error codes)
```

We've mostly kept them brief in other files:
```suggestion
    "F",       # pyflakes
    "I",       # isort
    "UP",      # pyupgrade
    "RUF100",  # Ban unused `# noqa` comments
    "PGH004",  # Ban blanket `# noqa` comments (only ignore specific error codes)
```